app/views/student_flashcards/show.html.erb

```diff
@@ -0,0 +1,16 @@
+show page for student flashcard
+
+<%= link_to 'Back', student_student_flashcard_sets_student_flashcards_path, class: "btn btn-outline-dark m-2" %>
+<div class="container mt-3">
+  <div class="quiz-question-card">
+   <div class="container">
```

Comment:
container is a bootstrap class
I wouldn't use it inside of quiz-question-card
I think you can just have a div, without any class